index.html

```diff
@@ -26,10 +27,9 @@
         </ul>
 
     </div>
-        
+
```

Comment:
The `header` should include your logo which should be a clickable link, so when visitors click on it, they are taken back to the top/home page. Also, the `header` should also contain a `nav` tag which will house the hamburger icon. Right now you are missing both of these requirements. Kindly implement them.